.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "astropy_helpers"]
```

Comment:
Why not just remove the entire file? In your branch here, I see an empty `.gitmodules` file.

Reply:
Removed, thanks.